routes/api.js

```diff
@@ -1,44 +1,34 @@
 'use strict';
 
-var apiConfig = require('../server/config').api;
-var COCKTAIL_PATH = apiConfig.base + apiConfig.cocktailPath;
-var dbOperations = require('../server/db-operations');
+const apiConfig = require('../server/config').api;
```

Comment:
Two notes here:
1.) Why are we using `require`, not `import from...`? ES6 modules are still in progress in Node; that's why we're still using `module.exports` instead of `export default`, too.
2.) Why not `const { apiConfig } = require('../server/config')`? Because object destructuring is also still in progress. Normally, though, we'd want to grab specific properties with the destructuring shorthand.